graphbolt/src/csc_sampling_graph.cc

```diff
@@ -781,10 +782,16 @@ inline int64_t LaborPick(
     std::iota(picked_data_ptr, picked_data_ptr + num_neighbors, offset);
     return num_neighbors;
   }
-  torch::Tensor heap_tensor = torch::empty({fanout * 2}, torch::kInt32);
+#define STACK_SIZE_LABOR 1024
```

Comment:
```suggestion
  static const int kLaborStackSize = 1024;
```

Reply:
I originally defined it as constexpr int, but msvc in CI doesn't support it properly. It is used as a template parameter for std::array so it needs to be defined at compile time.

Reply:
Weird, if that's the case, move the `#define` to the top, after L23. 

`#define kLaborStackSize  1024`